inject/configdriven/configdriven/src/main/java/io/helidon/inject/configdriven/ConfigDrivenInjectModule.java

```diff
@@ -16,12 +16,15 @@
 
 package io.helidon.inject.configdriven;
 
+import io.helidon.common.Weight;
+import io.helidon.common.Weighted;
 import io.helidon.inject.service.ModuleComponent;
 import io.helidon.inject.service.ServiceBinder;
 
 /**
  * An explicit and manually implemented module component, as we are doing a bit of unusual work with config bean registry.
  */
+@Weight(Weighted.DEFAULT_WEIGHT + 100)
```

Comment:
why is this not DEFAULT_WEIGHT - 1 like the others?

Reply:
because this one has to be the very first, as it is used by config driven (and also this is our own module that defines the config bean registry - if anybody wanted to use their own config bean registry, this module should not be on the classpath)

Reply:
The convention I saw throughout the rest of the code was to always use DEFAULT - 1 so that any implementation the customer provides naturally overrides our default choices.  I think that makes more sense.

Reply:
This service cannot be overridden by the customer and must be available in the registry before other services are registered. It does not work if the priority is default-1

Reply:
This is a requirement for the whole thing to work without designing new extension point for registry.
The modules are loaded sequentially depending on weight. ConifgDriven service activators expect this instance to be available in the registry, so it must have weight higher than them.
I am giving enough space for user ordering of config driven modules (and as we do not provide a mechanism to customize weight of a `ModuleComponent`, this should work just fine.